modules/rte_mis_lottery/src/Plugin/DevelGenerate/MiniNodeDevelGenerates.php

```diff
@@ -402,19 +435,20 @@ private function generateParagraph($paragraph_type = '', $data = []) {
         break;
 
       case 'field_school_preferences':
+        // $medium_options = ['hindi', 'english'];
```

Comment:
Please remove the commented code.

Reply:
This file is only for test purpose. We can ignore this for now.